views/auth/login/login.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../../assets/css/style.css" />
+    <link rel="stylesheet" href="../login/login.css" />
+    <title>JSMonk ecommerce website - Login</title>
+</head>
+
+<body>
+    <div id="main-container" class="main-container theme-light">
+        <!-- Header  -->
+        <div class="header mb-5 box-shadow-medium">
+            <div class="header-logo-wrapper px-5 py-2">
+                <div class="logo">
+                    <span class="logo-title">jsmonk ecomm</span>
+                    <span class="logo-subtitle">Best ecommerce website</span>
+                </div>
+            </div>
+        </div>
+        <div class="page">
+            <section class="auth-content m-5">
+                <div class="content-body box-shadow-medium p-5">
+                    <h2>Login</h2>
+                    <div class="m-5">
+                        <!-- Horizontal Input field -->
+                        <div class="form-group form-group-row">
+                            <label for="inputId">Email</label>
+                            <input type="email" id="inputId" placeholder="Email" class="input-control" />
```

Comment:
You could change the id of input of both email to something more relevant like email instead of inputId. same for password too